ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -188,25 +189,27 @@ final class RecoveringIndexerSpec
     }
 
     "report correct health status updates on failures and recoveries" in {
+      val logger = ContextualizedLogger.get(getClass)
       val healthStatusLogCapture = mutable.ArrayBuffer.empty[HealthStatus]
       val healthStatusRef = new AtomicReference[HealthStatus]()
 
-      val recoveringIndexer =
-        new RecoveringIndexer(
-          actorSystem.scheduler,
-          actorSystem.dispatcher,
-          someTimeout,
-          updateHealthStatus = healthStatus => {
-            healthStatusLogCapture += healthStatus
-            healthStatusRef.set(healthStatus)
-          },
-          () => healthStatusRef.get(),
-        )
+      val timeout = 100.millis
```

Comment:
timeout = 100.millis
someTimeout = 10.millis

Can we maybe just rename them to the timeout duration or something? So that you know what to expect when you see it used

Reply:
I copied the declaration to each test so it's close to the usage, and there's only one in scope at a time. I think this makes it much clearer. Let me know what you think.